packages/fxa-settings/src/pages/Signup/index.test.tsx

```diff
@@ -566,4 +566,113 @@ describe('Signup page', () => {
+      //
+      // Example implementation
+      // renderWithLocalizationProvider(
+      //   <Subject
+      //     {...{
+      //       queryParams: {
+      //         email: '   foo@bar.com',
+      //         emailFromContent: 'true',
+      //       },
+      //     }}
+      //   />
+      // );
+      // const email = screen.getByTestId('hidden-email');
+      // expect(email.getAttribute('value')).toEqual('foo@bar.com');
+    });
+
+    it('checks coppa is empty', () => {
+      renderWithLocalizationProvider(
+        <Subject
+          {...{
+            queryParams: {
+              email: 'foo@bar.com',
+              emailFromContent: 'true',
+            },
+          }}
+        />
+      );
+
+      const newPasswordInput = screen.getByTestId('new-password-input-field');
+      const verifyPasswordInput = screen.getByTestId(
+        'verify-password-input-field'
+      );
+      const ageInput = screen.getByTestId('age-input-field');
+      const createAccountButton = screen.getByText('Create account');
+
+      fireEvent.change(newPasswordInput, {
+        target: { value: 'bar12345' },
+      });
+      fireEvent.change(verifyPasswordInput, {
+        target: { value: 'bar12345' },
+      });
+      fireEvent.focus(ageInput);
+      fireEvent.blur(ageInput);
+      createAccountButton.click();
+
+      expect(
+        screen.getByText('You must enter your age to sign up')
+      ).toBeInTheDocument();
+      expect(createAccountButton).toBeDisabled();
+
+      // TBD: I noticed that you can enter non-numeric values for age and doing so results in no error, but
+      // the Create account button also doesn't become enabled. I'm not sure if we want to prevent non-numeric
+      // input from being entered into this box or not... The old version does not allow entering non numeric
+      // values.
+      //
+      // fireEvent.change(ageInput, { target: { value: 'aa' } });
+      // expect(
+      //   screen.getByText('You must enter your age to sign up')
+      // ).toBeInTheDocument();
+    });
+
+    it('shows error for non matching passwords', () => {
+      renderWithLocalizationProvider(
+        <Subject
+          {...{
+            queryParams: {
+              email: 'foo@bar.com',
+              emailFromContent: 'true',
+            },
+          }}
+        />
+      );
+
+      fireEvent.change(screen.getByTestId('new-password-input-field'), {
+        target: { value: 'bar12345' },
+      });
+      fireEvent.change(screen.getByTestId('verify-password-input-field'), {
+        target: { value: 'bar12346' },
+      });
+      fireEvent.blur(screen.getByTestId('verify-password-input-field'));
+      expect(screen.getByText('Passwords do not match')).toBeInTheDocument();
+    });
+
+    it('signup, verify and sign out of two accounts, all in the same tab, then sign in to the first account', () => {
+      // TBD: This test probably is no longer applicable. It was checking that tabs change  after signing in
```

Comment:
Was this intended to check that the signin shows the right email from localStorage? In any case, if we're keeping this it should be a Playwright test because of the navigation between signup, settings, sign in. Agree that the following one is probably no longer applicable because we aren't doing that polling anymore.